src/modules/LanUdevAuto.rb

```diff
@@ -180,7 +180,7 @@ def Import(settings)
           createUdevFromIfaceName(Ops.get_list(settings, "interfaces", []))
         )
       else
-        @udev_rules = Ops.get_list(settings, "net-udev", [])
+        @udev_rules = settings["net-udev"] || []
```

Comment:
actual fix is in the other file in the commit. I was also checking if an expected default value is set properly everywhere, so I turned it from converted code into ruby here.